### PR TITLE
[NETBEANS-5678] Use the URL of the forked repository of the Skeleton Generator

### PR DESCRIPTION
**Pull request by junichi11**
https://issues.apache.org/jira/browse/NETBEANS-5678

- The original PHPUnit skeleton generator has been abandoned

The forked repo: https://github.com/VitexSoftware/phpunit-skeleton-generator

netbeans-5678-phpunit-options

### CODE REVIEW

File: php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties [hunk 1]
```diff
@@ -59,6 +59,7 @@ PhpUnitOptionsPanel.phpUnitInfoLabel.AccessibleContext.accessibleName=PHPUnit cl
 PhpUnitOptionsPanel.noteLabel.AccessibleContext.accessibleName=Note:
 PhpUnitOptionsPanel.skelGenLabel.text=&Skeleton Generator Script:
 PhpUnitOptionsPanel.phpUnitLearnMoreLabel.text=<html><a href="#">Learn more about PHPUnit</a></html>
-PhpUnitOptionsPanel.skelGenLearnMoreLabel.text=<html><a href="#">Learn more about Skeleton Generator</a></html>
+PhpUnitOptionsPanel.skelGenLearnMoreLabel.text=<html><a href="#">Learn more about Skeleton Generator</a> (forked repo.)</html>
 PhpUnitOptionsPanel.skelGenBrowseButton.text=&Browse...
 PhpUnitOptionsPanel.skelGenSearchButton.text=S&earch...
+PhpUnitOptionsPanel.skelGenAbandonedLabel.text=<html><b>The original generator has been abandoned.</b></html>
```
Comment (tmysik) [inline php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties:65]: Nitpick: perhaps "The original [project]"?

**Review by tmysik — APPROVED** [review]

**Comment by tmysik** [comment]
@junichi11 Please, let me know - again, heh - whether this PR can be merged. Thanks!

File: php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties (hunk 1 above)
Comment (junichi11) [reply php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties:65]: Will fix ("The original project has been abandoned."). Thanks!

**Comment by junichi11** [comment]
@tmysik We can merge it :)

File: php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties (hunk 1 above)
Comment (junichi11) [reply php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties]: Fixed: https://github.com/apache/netbeans/compare/5b834cfeff13ca3279108fa7f0e0cba8448f5d91..2f71bb9cbc2e065360a210d7ef32caa98a0278e9

**Comment by tmysik** [comment]
@junichi11 

Will wait for the other reviewers and merge it later. Thanks!

**Comment by junichi11** [comment]
@tmysik OK. Just in case, I added other reviewers. (Maybe, the feature freeze date is tomorrow(7/15).) Thanks!

**Comment by tmysik** [comment]
Not a big change, merging.

**Pull request merged**